server/models/articles.js

```diff
@@ -0,0 +1,64 @@
+const getArticleModel = (sequelize, DataTypes) => {
+  const flags = {
+    timestamps: false,
+    freezeTableName: true
+  };
+  const articleSchema = {
+    id: {
+      type: DataTypes.INTEGER,
```

Comment:
I do not agree with this id. So what you are telling me in a top database you are going to have your users labelled sequentially. This is just light work for hackers

Reply:
Please remove this model and let @mrkingz  raise a PR for it with my changes